api/mutations/community/updateAdministratorEmail.js

```diff
@@ -24,6 +24,14 @@ export default (
     return new UserError('Please enter a working email address');
   }
 
+  const permissions = await loaders.userPermissionsInCommunity.load([
+    currentUser.id,
+    id,
+  ]);
+
+  if (!permissions || !permissions.isOwner)
```

Comment:
I think this is a good place to use the `userCanAdministerCommunity` helper in `utils/permissions` :)